README.md

```diff
@@ -31,10 +31,11 @@ After completing this code pattern, the user will learn how to:
 
 ![](doc/source/images/architecture.png)
 
-1. The user provides the client with an SMS message to be analyzed.
-2. The client sends the SMS to Watson NLU for analysis, specifying which machine learning WKS model to use.
-3. Watson NLU extracts the domain specific entities and returns the results to the client.
-4. The client renders the results to the user.
+1. Corpus and type files are loaded into Watson Knowledge Studio.
+1. A user generates a model by training and evaluating data.
+1. The WKS model is deployed to Watson NLU.
+1. A user provides an SMS message to the app that is to be analyzed.
+1. The SMS message is analyzed by Watson NLU for processing and domain specific based on the WKS model are returned.
```

Comment:
Seems like a word is missing. I'll guess s/specific based/specific results based/

Reply:
Need numbers for each item. 
And for content, might I suggest...
1) Load type system and corpus files into Watson Knowledge Studio
2) ..
3) ...
4) A user provides an SMS message to the app for analysis
5) The SMS message is analyzed by Watson NLS for processing and returns extracted domain specific entities based on the WKS model..

Reply:
diagram looks good

Reply:
you can keep the numbers like that in markdown, it'll render properly in html and in the github viewer